### DISCUSSION TITLE
xml files

### DISCUSSION THREAD

**Discussion by persianpros** [ctlcltd/e2-sat-editor#7] [Ideas]
Hi from Open Vision team :)

Just wanted to let you know about https://github.com/OpenVisionE2/openvision-xml

You may want to add xml updates for your users :+1:

**Comment by persianpros** [comment]
Also we provide some settings https://github.com/OpenVisionE2?q=settings&type=all&language=&sort=

**Comment by persianpros** [comment]
I can test your builds also on Ubuntu arm64, win11 arm64 and Mac M1 if you want to support arm64 as well.

**Comment by ctlcltd** [comment]
@persianpros Nice work, congratulations! I will consider a feature to update satellites.xml and the others.

The intent would be to create a plugin manager, but it will take some time. I've started a very embryonic integration with Swig (https://www.swig.org) but it's not something yet usable.

I exclude downloading settings from remote sources at this time. It requires me to be more responsible with users.

**Comment by ctlcltd** [comment]
> I can test your builds also on Ubuntu arm64, win11 arm64 and Mac M1 if you want to support arm64 as well.

👍 It should work fine on other architectures. If you try let me know, thanks

**Comment by persianpros** [comment]
> > I can test your builds also on Ubuntu arm64, win11 arm64 and Mac M1 if you want to support arm64 as well.
> 
> +1 It should work fine on other architectures. If you try let me know, thanks

What I mean is native arm64 build if you want to provide it :+1:

**Comment by ctlcltd** [comment]
@persianpros I understand, and I thank you for your availability.
It's generally not a good idea to provide builds from strangers. I would like automated builds in the near future. They are currently deployed manually without signatures. On modern Windows the price for signatures is out of reach, and there is a scoring mechanism, on macOS I have to think about it.
